pyphare/pyphare/simulator/monitoring.py

```
@@ -13,6 +13,29 @@
 logger = getLogger(__name__)
 
 
+@dataclass
+class MonitoringOptions:
+    interval: int = 0
+    rank_modulo: int = 1  # all ranks
+
+    def __post_init__(self):
+        if rank_modulo == 0:
```

Ruff (0.15.15)

[error] Undefined name `rank_modulo`

(F821)